research-software-directory/RSD-as-a-service
# Rename Helmholtz AAI to Helmholtz ID

# Rename Helmholtz AAI to Helmholtz ID

Changes proposed in this pull request:

* renames and replaces all occurrences of "Helmholtz AAI" to "Helmholtz ID" which is the new branding for the Helmholtz IdP

How to test:

* rename environment variables from `HELMHOLTZAAI_*` to `HELMHOLTZID_*`
* add `HELMHOLTZID` to the available authentication methods
* `docker compose build --parallel && docker compose up`
* login via Helmholtz ID should work as usual

PR Checklist:
